### ISSUE TITLE
0.5.2 (Final) does not work with Windows Vista

### ISSUE THREAD

**Issue by 4194304** [Ingan121/BasicThemer2#25]
When I attempt to run 0.5.2 (Final) on Vista it gives an error:
![image](https://user-images.githubusercontent.com/130800536/235319896-46dfe968-8f53-4776-97f5-ef93b83820f0.png)
Hopefully there is a fix for this

**Comment by retro925** [comment]
cuz basic theme is already on windows vista dummy

**Comment by 4194304** [comment]
No, this is meant to work on Vista, and also this project is about enabling the basic theme without disabling composition, which using the basic theme on Vista does disable composition
